test/ContractGenerator.Tests/ContractGeneratorTests.cs

```diff
@@ -4,7 +4,7 @@ public class ContractGeneratorTests
 {
     private static FileStream GetInputStream(string testcaseName)
     {
-        return File.Open($"testcases/{testcaseName}/descriptor.bin", FileMode.Open);
+        return File.Open($"testcases/{testcaseName}/descriptor.bin", FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```

Comment:
ah i guess this mode is to allow multiple tests to read the file at the same-time? tho wonder why write-access would be required by the tests. 🤔